src/dispatch/__tests__/EnhancedAuthentication.test.ts

```diff
@@ -31,18 +28,6 @@ describe('EnhancedAuthentication tests', () => {
             sessionToken: 'z',
             expiration: new Date(Date.now() + 3600 * 1000)
```

Comment:
nit: is GetCredentialsForIdentity supposed to return `secretKey` instead of `secretAccessKey`?

https://docs.aws.amazon.com/cognitoidentity/latest/APIReference/API_GetCredentialsForIdentity.html

Reply:
The `CognitoIdentityClient`, which is implemented by the RUM web client (recall we don't use the AWS SDK here), returns `secretAccessKey`. This is by design, because it returns a `Credential` object, which has the property `secretAccessKey`.

One could make the argument that the `CognitoIdentityClient` should have behavior which is consistent with the actual Cognito client. However, I've altered this behavior for the sake of maintainability and reducing the size of the web client bundle.